pandas/io/json/_normalize.py

```diff
@@ -286,12 +290,15 @@ def _recursive_extract(data, path, seen_meta, level=0):
         else:
             for obj in data:
                 recs = _pull_field(obj, path[0])
-                recs = [
-                    nested_to_record(r, sep=sep, max_level=max_level)
-                    if isinstance(r, dict)
-                    else r
-                    for r in recs
-                ]
+                if not is_scalar(recs):
```

Comment:
it might be better for _pull_field to actually *always* return a list rather than a scalar, no? then you don't need this at all

Reply:
Good point. Will do.